repo/gromacs/application.py

```diff
@@ -0,0 +1,181 @@
+    workload_variable('input_path', default='{HECBioSim}/HECBioSim/{type}/benchmark.tpr',
+                      description='Input path for hecbiosim',
+                      workload='hecbiosim')
+    workload_variable('input_path', default='{BenchPEP}/benchPEP.tpr',
+                      description='Input path for Bench PEP workload',
+                      workload='benchpep')
+    workload_variable('input_path', default='{BenchMEM}/benchMEM.tpr',
+                      description='Input path for Bench MEM workload',
+                      workload='benchmem')
+    workload_variable('input_path', default='{BenchRIB}/benchRIB.tpr',
+                      description='Input path for Bench RIB workload',
+                      workload='benchrib')
+    workload_variable('input_path', default='{BenchPEP_h}/benchPEP-h.tpr',
+                      description='Input path for Bench PEP-h workload',
+                      workload='benchpep_h')
+    workload_variable('type', default='rf_nvt',
+                      description='Workload type for JCP_benchmarks',
+                      workload='stmv_rf')
+    workload_variable('type', default='pme_nvt',
+                      description='Workload type for JCP_benchmarks',
+                      workload='stmv_pme')
+    workload_variable('type', default='grompp',
+                      description='Workload type for JCP_benchmarks',
+                      workloads=['ion_channel', 'rnase_cubic'])
+    workload_variable('input_path', default='{JCP_benchmarks}/stmv',
+                      description='Input path for JCP_benchmark {workload_name}',
+                      workloads=['stmv_rf', 'stmv_pme'])
+    workload_variable('input_path', default='{JCP_benchmarks}/{workload_name}',
+                      description='Input path for JCP_benchmark {workload_name}',
+                      workloads=['ion_channel', 'rnase_cubic'])
+
+    log_str = os.path.join(Expander.expansion_str('experiment_run_dir'),
+                           'md.log')
+
+    figure_of_merit('Core Time', log_file=log_str,
+                    fom_regex=r'\s+Time:\s+(?P<core_time>[0-9]+\.[0-9]+).*',
+                    group_name='core_time', units='s')
+
+    figure_of_merit('Wall Time', log_file=log_str,
+                    fom_regex=r'\s+Time:\s+[0-9]+\.[0-9]+\s+' +
+                              r'(?P<wall_time>[0-9]+\.[0-9]+).*',
+                    group_name='wall_time', units='s')
+
+    figure_of_merit('Percent Core Time', log_file=log_str,
+                    fom_regex=r'\s+Time:\s+[0-9]+\.[0-9]+\s+[0-9]+\.[0-9]+\s+' +
+                              r'(?P<perc_core_time>[0-9]+\.[0-9]+).*',
+                    group_name='perc_core_time', units='%')
+
+    figure_of_merit('Nanosecs per day', log_file=log_str,
+                    fom_regex=r'Performance:\s+' +
+                              r'(?P<ns_per_day>[0-9]+\.[0-9]+).*',
+                    group_name='ns_per_day', units='ns/day')
```

Comment:
Note that this is the only FOM that really matters.

Reply:
Do you think it would be less confusing if we remove the rest of the FOMs?